grpc-example/grpc-example-java/hello-proxy-impl/src/main/resources/application.conf

```diff
@@ -9,12 +9,12 @@ akka.grpc.client {
     service-discovery {
       mechanism = "lagom-dev-mode"
       service-name = "helloworld.GreeterService"
-      port-name = "https"
+      port-name = "http"
     }
     override-authority = "localhost"
     deadline = 5s
     connection-attempts = 5
+    use-tls = false
   }
 
 }
```

Comment:
Isn't this necessary on the `-scala` sample aswell?

Reply:
It was already done, programatically. 🤷

Reply:
Aaaah! You're right! I remember that code now.